.zuul.yaml

```diff
@@ -30,7 +30,6 @@
       jobs:
         - cloud-provider-openstack-format:
             irrelevant-files:
-              - ^docs/.*$
```

Comment:
Why this line is removed?

Reply:
docs only changes won't run format check now
we need remove it so that docs change trigger format test as well

Reply:
So you mean cloud-provider-openstack-format job also checks the doc?

Reply:
https://logs.openlabtesting.org/logs/40/1240/9e18b248d65751bef2e47b95968c05154a960b4c/cloud-provider-openstack-format/cloud-provider-openstack-format/4567a60/job-output.txt.gz

search `developement` keyword


2020-10-13 06:31:41.555647 | ubuntu-xenial-ut | docs/cinder-csi-plugin/using-cinder-csi-plugin.md:18:11: "Developement" is a misspelling of "Development"
2020-10-13 06:31:41.555754 | ubuntu-xenial-ut | docs/cinder-csi-plugin/using-cinder-csi-plugin.md:18:32: "developement" is a misspelling of "development"
2020-10-13 06:31:41.555792 | ubuntu-xenial-ut | docs/cinder-csi-plugin/using-cinder-csi-plugin.md:165:9: "Developement" is a misspelling of "Development"